demos/baxtergv6_apc2016/README.md

```diff
@@ -0,0 +1,44 @@
+# baxtergv6_apc2016
+
+
+Execute mimic APC2016 tasks with ARC2017 robot system (baxter with gripper-v6).
+
+
+## Installation
+
+1. [Install jsk_apc](https://github.com/start-jsk/jsk_apc#installation).
+2. `rosrun baxtergv6_apc2016 install_data`
+
+
+## Pick Task Execution
+
+```bash
+roslaunch baxtergv6_apc2016 baxter.launch
+roslaunch baxtergv6_apc2016 setup_for_pick.launch
+roslaunch baxtergv6_apc2016 pick.launch json_dir:=$(rospack find baxtergv6_apc2016)/data/json/pick1
+```
+
+
+## Stow Task Execution
+
+```bash
+roslaunch baxtergv6_apc2016 baxter.launch
+roslaunch baxtergv6_apc2016 setup_for_stow.launch
+roslaunch baxtergv6_apc2016 stow.launch json_dir:=$(rospack find baxtergv6_apc2016)/data/json/stow1
+```
+
+
+## Citation
+
+```
+@article{Hasegawa_2019jrm,
```

Comment:
please add link to your publication https://www.fujipress.jp/jrm/rb/robot003100020289/ ?

Reply:
Thank you, I added the link